core/src/main/java/com/dtolabs/rundeck/core/config/RundeckConfigBase.java

```diff
@@ -461,6 +460,11 @@ public static class RundeckFeatureConfig {
         Enabled enhancedJobTakeoverQuery = new Enabled();
         Enabled publicKeysDownload = new Enabled();
         Enabled guiHideRoiInstructions = new Enabled();
+        Enabled defaultExecutionCleanup = new Enabled();
```

Comment:
I think we can move these to a separate class like 
````
@Data
public static class RundeckExecutionCleanupConfig {
        Integer defaultExecutionDaysToKeep;
        Integer defaultExecutionMinimumToKeep;
        Integer defaultExecutionMaximumDeletionSize;
        String defaultExecutionCleanupScheduleCron;
}
````

You can set this as a field in RundeckConfigBase 

````
RundeckExecutionCleanupConfig executionCleanupConfig
````

finally the config value should be

````
rundeck.executionCleanupConfig.defaultExecutionDaysToKeep=xxx
rundeck.executionCleanupConfig.defaultExecutionMinimumToKeep=xxx
rundeck.executionCleanupConfig.defaultExecutionMaximumDeletionSize=xxx
rundeck.executionCleanupConfig.defaultExecutionCleanupScheduleCron=xxx
````